src/api/flaskr/service/study/funcs.py

```diff
@@ -308,7 +308,7 @@ def get_lesson_tree_to_study(
 ) -> AICourseDTO:
     return run_with_redis(
         app,
-        app.config.get("REDIS_KEY_PRRFIX") + "::get_lesson_tree_to_study:" + user_id,
+        app.config.get("REDIS_KEY_PREFIX") + ":get_lesson_tree_to_study:" + user_id,
```

Comment:
@yfge @aichy126 I can find out who wrote this line of code through git log. Please review the removed `:`.